script/Ciel/Day 7/07_01_CIEL07_6.txt

```diff
@@ -1035,28 +1073,28 @@ Though this job is over with, I don't feel much of a sense of achievement.
 [sha:fcf6964d430a0953cb2cf1a03477f99fdf36a13f]{
 -- Page 572, Offset 16182.
 -- これでロアの残党狩りが終わった訳でもないし、なにより、このマンションの処遇はどうしたものか。
-I mean, it's not like the hunt for the Roa remnants is over,
+I mean, it's not like the hunt for Roa's remaining offspring is over,
  and more importantly,
- how are we supposed to take care of this apartment complex?
+ what are we supposed to do about this apartment complex?
 }
 [sha:ea63268fd6a858f976271777d952dad7e9221248]{
 -- Page 573, Offset 16183.
 -- 「どうするんだアルクェイド。他に吸血鬼がいないとしても、ここはねぐらの一つなんだろ？　もしかしたら中に隠れているヤツがいるかもしれないし、念のために調べるか？」
-"What will you do, Arcueid?
- Even if there aren't any other vampires here,
- this is still one of their nests, isn't it?
- There might still be one of them hiding inside,
- so, wanna investigate just in case?"
+"What do we do now, Arcueid?
+ It doesn't seem like there are any more vampires here,
+ but this is still one of their nests, right?
+ There might be more of them hiding inside,
+ so should we investigate just to be on the safe side?"
 }
 [sha:8a4bb2ad5aadf0d0568ca219b67e5631a43f7ca1]{
 -- Page 574, Offset 16184.
 -- 「もちろん、しっかり後始末はするわ。
-"I'll take care of it, of course.
+"We make sure there are no loose ends, of course.
```

Comment:
This one feel's like it shouldn't be using the inclusive 'we'

Reply:
This is for conversational coherence - 
> What do we do now, Arcueid? [...]
> "We make sure there are no loose ends

He's obviously not helping, but he's party. I don't think `we` is crazy here.

Reply:
I understand wanting to render the question with 'we', because that's probably how Shiki would say such a thing in English, but its quite clear from Arcueid's answer that she's the one who'll take care of this particular thing (and its what happens, too). It feels a bit bizarre with 'we' here, in that sense.

So either you change the initial part to something like "So, what happens next, Arcueid?" so you can avoid the we to begin with, or you just ignore keeping them consistent (it wouldn't be a crazy notion exactly).